api/Element.json

```diff
@@ -2231,6 +2231,38 @@
           }
         }
       },
+      "attributeStyleMap": {
```

Comment:
This is already on HTMLElement. Where should it be?

Reply:
Found this is in the `custom-idl` with a link to a resolved BCD issue, so I'll undo this!